apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -109,7 +109,7 @@ export default class DetailViewContents extends React.Component {
           >
             {
               STATUSES.map((status, i) => (
-                <option value={status.toLowerCase()} key={i}>
+                <option value={status.toLowerCase().replace(/ /g, '_')} key={i}>
```

Comment:
You could just use `_.snakeCase(status)`

Reply:
oh good point! they have actually decided to switch the status name to "Interview" so this is coming out in this PR: https://github.com/code-dot-org/code-dot-org/pull/18887